src/OptimizerChainFactory.php

```diff
@@ -15,9 +15,11 @@ public static function create(array $config = []): OptimizerChain
     {
         $jpegQuality = '--max=85';
         $pngQuality = '--quality=85';
+        $webpQuality = '-q 80';
         if (isset($config['quality'])) {
             $jpegQuality = '--max='.$config['quality'];
             $pngQuality = '--quality='.$config['quality'];
+            $webpQuality = '-q 80'.$config['quality'];
```

Comment:
```suggestion
            $webpQuality = '-q '.$config['quality'];
```

Reply:
thanks for noticing my copy&paste mistake 🙈